api/sendfild/main.go

```diff
@@ -0,0 +1,210 @@
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net"
+	"time"
+
+	logging "github.com/ipfs/go-log/v2"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"github.com/textileio/go-threads/util"
+	"github.com/textileio/powergate/v2/lotus"
+	"github.com/textileio/textile/v2/api/sendfild/service"
+	"github.com/textileio/textile/v2/cmd"
+)
+
+const daemonName = "sendfild"
+
+var (
+	log = logging.Logger(daemonName)
+
+	config = &cmd.Config{
+		Viper: viper.New(),
+		Dir:   "." + daemonName,
+		Name:  "config",
+		Flags: map[string]cmd.Flag{
+			"debug": {
+				Key:      "debug",
+				DefValue: false,
+			},
+			"logFile": {
+				Key:      "log_file",
+				DefValue: "", // no log file
+			},
+			"listenAddr": {
+				Key:      "listen_addr",
+				DefValue: "127.0.0.1:5000",
+			},
+			"lotusAddr": {
+				Key:      "lotus_addr",
+				DefValue: "127.0.0.1:7777",
+			},
```

Comment:
Notice these are now just plain `host:port`, no multiaddr. This depends on an [open Powergate PR](https://github.com/textileio/powergate/pull/791).